main/templates/main/index.html

```diff
@@ -1,14 +1,45 @@
 {% extends 'base.html' %}
+{% load static %}
+
 {% block title %} Tutor Online {% endblock %}
+
+{% block links %}
+<link rel="stylesheet" href="{% static 'css/main.css' %}">
+{% endblock %}
+
 {% block content %}
-<div class="container">
-  <div class="jumbotron">
-    <h1>Welcome to Tutor Online !</h1>
-    {% if user.is_authenticated %}
-        <h2>Hello {{ user.first_name }} {{ user.last_name }}</h2>
-    {% else %}
-        <h2>Register or Login if you'd like to</h2>
-    {% endif %}
+<style>
+  html {
+    overflow: scroll;
+    overflow-x: hidden;
+}
+::-webkit-scrollbar {
+    width: 0px;  /* Remove scrollbar space */
+    background: transparent;  /* Optional: just make scrollbar invisible */
+}
+</style>
+
+<div class="hero is-fullheight is-light">
+  <div class="hero-header text-center mt-5">
+    <p class="is-size-1-desktop is-size-2-tablet is-size-2-mobile animated fast fadeInUp">Welcome to Tutor Online !</p>
+  </div>
+  <div class="hero-body justify-content-center animated slow fadeInUp">
+    <div class="container text-center">
+      {% if user.is_authenticated %}
+          <p class="is-size-4-desktop is-size-5-tablet is-size-5-mobile">Hello, {{ user.first_name }} {{ user.last_name }}!</p> <br>
+          <button class="button is-link">Go to dashboard <i class="ml-5 fa fa-arrow-right"></i></button>
```

Comment:
buat student lebih penting buat go to lesson, sama buat teacher lebih penting buat go profile (buat add portfolio)

Reply:
gimana caranya tau yang login itu student atau teacher?

Reply:
user punya atribut boolean `is_teacher` .. jadi bisa langsung dicek aja

Reply:
ok 👍